.github/workflows/extbuild.yml

```diff
@@ -74,28 +48,27 @@ jobs:
           parallelio_version: ${{ env.ParallelIO_VERSION }}
           enable_fortran: True
           install_prefix: $HOME/pio
-      - name: Cache ESMF
-        id: cache-esmf
-        uses: actions/cache@v3
-        with:
-          path: ~/ESMF
-          key: ${{ runner.os }}-${{ env.ESMF_VERSION }}-ESMF2
-      - name: Build ESMF
-        if: steps.cache-esmf.outputs.cache-hit != 'true'
-        uses: ./.github/actions/buildesmf
+      - name: Install ESMF
+        uses: esmf-org/install-esmf-action@v1
+        env:
+          ESMF_COMPILER: gfortran
+          ESMF_BOPT: g
+          ESMF_COMM: openmpi
+          ESMF_NETCDF: nc-config
+          ESMF_PNETCDF: pnetcdf-config
+          ESMF_INSTALL_PREFIX: /home/runner/ESMF
```

Comment:
You should still be able to use $HOME
`ESMF_INSTALL_PREFIX: $HOME/ESMF`

Reply:
done